manifests/init.pp

```diff
@@ -482,6 +482,11 @@
 #
 # $server_allow_header_cert_info::          Enable client authentication over HTTP Headers
 #                                           Defaults to false, is also activated by the $server_http setting
+#
+# $server_web_idle_timeout::                Time in ms that Jetty allows a socket to be idle, after processing has
+#                                           completed.
+#                                           Defaults to 30000, using the Jetty default of 30s
```

Comment:
The default according to https://docs.puppet.com/puppetserver/latest/config_file_puppetserver.html is 20 minutes (1,200,000 milliseconds), should this be the same?